.cirrus.yml

```diff
@@ -68,14 +68,22 @@ task:
 # Tasks for Linux aarch64 native builds 
 task:
   matrix:
-    - name: Linux aarch64
+    - name: Linux aarch64 glibc
       arm_container:
         image: rust:1.69.0
         cpu: 1
       depends_on:
         - FreeBSD 14 amd64 & i686
```

Comment:
BTW, is it still necessary to make Linux native builds depend on the FreeBSD test? This dramatically slows down our CI

Reply:
> And while we don't have a full month worth of data yet

Right, I prefer to do this until we know the appropriate usage of a month, just in case